.github/workflows/switch-region.yml

```diff
@@ -0,0 +1,66 @@
+name: Switch region to eu-central-1
+
+on:
+  workflow_dispatch: # Manual trigger
+    inputs:
+      environment:
+        description: 'Choose environment'
+        required: true
+        default: prod/eu-south-1
```

Comment:
It does not make to much sense this input parameter. 
If I'm going to ask you to chose between one and one what would you chose? :-)